superset/assets/src/dashboard/stylesheets/grid.less

```diff
@@ -17,6 +17,11 @@
   display: flex;
   flex-direction: column;
   margin-bottom: 100px;
+  padding-top: 16px;
```

Comment:
hmm I fixed the extra padding already which was due to an empty droptarget, can you remove this and the `padding-top` line below?

Reply:
removed.